<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>661</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>661</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Track Selection</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="trackSelection">
        <item>
         <property name="text">
          <string>Silverstone</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Monaco</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Montreal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Shanghai</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Singapore</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Belgium</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="trackViewer">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>475</width>
       <height>200</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>661</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="videoButton">
       <property name="text">
        <string>Open Video/Image</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="csvButton">
       <property name="text">
        <string>Open Telemetry File</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="videoPath">
       <property name="text">
        <string>Select File Location</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="csvPath">
       <property name="text">
        <string>Select File Location (.csv)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>390</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="runvideoButton">
       <property name="text">
        <string>Video Mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="runButton">
       <property name="text">
        <string>Image Mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuF1_Trainning">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAbout_this_project"/>
   </widget>
   <addaction name="menuF1_Trainning"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout_this_project">
   <property name="text">
    <string>About this project</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
